extensions/amp-story-dev-tools/0.1/amp-story-dev-tools.css

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+html, body {
+  height: 100% !important;
+}
+
+amp-story-dev-tools {
+  width: 100% !important;
+  height: 100% !important;
+  box-sizing: border-box !important;
+  background-color: #121212 !important;
+  font-size: 16px !important;
+  font-family: 'Poppins', sans-serif !important;
+  color: white !important;
+  padding: 30px 10% !important;
+  text-align: center !important;
+}
+
+amp-story[standalone] {
+  display: none !important;
```

Comment:
This should only apply when there's a class or something, not whenever this CSS file is loaded. Same for the `html, body height 100%`

Reply:
Ok so in amp-story I assume